uber/config.py

```diff
@@ -160,14 +167,29 @@ def PREREG_DONATION_TIERS(self):
         return dict(self.PREREG_DONATION_OPTS)
 
     @property
-    def SUPPORTERS_ENABLED(self):
-        return self.SUPPORTER_LEVEL in self.PREREG_DONATION_TIERS
-
-    @property
```

Comment:
That may be why we originally added it, but I think it's no longer used even for that, because we're currently doing different if-else checks which don't involve this property.